2fa-api/handlers/user.js

```diff
@@ -0,0 +1,9 @@
+const user = function (req, res) {
+  try {
+    return res.status(200).json({ loggedIn: req.session.loggedIn });
```

Comment:
what does the backend need sessions for? is handling state in this way the right approach? can you foresee anything wrong with this?

Reply:
In this case its to track and validate the user between requests using session cookies. I guess the risk here is that the cookie is stolen and a bad actor validates themselves as a legit user. In terms of handling state here, I should create some sort of middleware to validate the user.

Reply:
the backend should be `stateless`. what if I deploy my service? all of a sudden my users are all logged out. a better approach:

session cookies on front end
an independent cache that a stateless backend service can call to determine if the session id is still valid

Reply:
also, how about jwts?